src/main/java/com/gdsc/boilerplate/springboot/payment/momo/shared/utils/Execute.java

```diff
@@ -15,17 +15,17 @@ public class Execute {
     public HttpResponse sendToMoMo(String endpoint, String payload) {
 
         try {
-
+            LogUtils.init();
```

Comment:
không ý tôi là dùng anotation slf4j á ông chớ LogUtils xoá file ni cũng được tại mình tự handle log theo ý mình, bên mình đang dùng cái anotation slf4j á nên dùng chung lun cho nó giống format nhau á ☝️👍👍👍👍👍👍!!
reference: [log in spring boot](https://www.baeldung.com/spring-boot-logging) (mục 7 nha 😉)

Reply:
đã update